ar-AA12uTHL.url

```diff
@@ -0,0 +1,2 @@
+[InternetShortcut]
```

Comment:
This file and a few others (`excel_override_poc` folder) shouldn't have been added here!
Remove all of them from this branch.

You can try using:
`git rm <file-name>` for a single file, or:
`git rm -r <folder-name>` to delete a folder from git.

Also, you can check if you are accidently adding files which you shouldn't add by using
`git status`
before or after using `git add`